src/token/Token.sol

```diff
@@ -403,6 +417,46 @@ contract Token is IToken, UUPS, Ownable, ReentrancyGuard, ERC721Votes, TokenStor
         return super.owner();
     }
 
+    /// @notice Update minters
+    /// @param _minters Array of addresses to add or remove as minters
+    /// @param _isMinter Array of booleans indicating whether the address is a minter
+    function updateMinters(address[] calldata _minters, bool[] calldata _isMinter) external onlyOwner {
```

Comment:
Thoughts on array of structs vs. two arrays here?

Reply:
+1, would solve the array length checking

Reply:
sure